src/lib.rs

```diff
@@ -496,6 +563,13 @@ impl Tag {
                 inner.remove_entries(&"ARTIST".into());
                 inner.add_one("ARTIST".into(), artist.into());
             }
+            Self::OggTag { inner } => {
+                inner.comments.remove("ARTIST");
+                inner.comments.insert(
+                    "ARTIST".into(),
+                    artist.split("; ").map(ToString::to_string).collect(),
```

Comment:
I'm not sure we want to make the assumption that multiple artists will be separated by semicolons -- different programs may handle it a different way. Does the ogg specification not allow duplicate tags?